test/invariant/handlers/HandlerBase.sol

```diff
@@ -15,6 +15,8 @@ contract HandlerBase is CommonBase, StdCheatsSafe, StdUtils {
 
     MockERC20[3] public assets;
 
+    uint256 public constant TRILLION = 1e12;
```

Comment:
It's a bit odd to have this as a constant variable in the base. This is not being reused in the sub-classes except for 1 and it's not an arbitrary value like `NUM_TOKENS` or something. I would just use 1e12 in `LpHandler` directly, and one million like you are in the `TransferHandler`.

Reply:
Updated

Reply:
Still need to delete the constant.